setup.sh

```diff
@@ -91,6 +93,8 @@ CHANNEL="${ARR_CHANNEL[0]}"
 [[ -z $ARCH ]] && ARCH=x64
 [[ -z $CACHE_PATH ]] && CACHE_PATH="$RUNNER_TEMP/flutter/:channel:-:version:-:arch:"
 [[ -z $CACHE_KEY ]] && CACHE_KEY="flutter-:os:-:channel:-:version:-:arch:-:hash:"
+[[ -z $PUB_CACHE_KEY ]] && PUB_CACHE_KEY="flutter-pub-:os:-:channel:-:version:-:arch:-:hash:"
+[[ -z $PUB_CACHE ]] && PUB_CACHE="$HOME/.pub-cache"
```

Comment:
Did you mean `PUB_CACHE_PATH`?

Reply:
@vladimir-goldobin No, in this case it's `PUB_CACHE` because it uses the environment variable that `dart` reads. I can add a comment. :)